src/subscription_manager/scripts/rhsmcertd_worker.py

```diff
@@ -170,8 +170,33 @@ def _auto_register_wait() -> None:
     if cfg.get("rhsmcertd", "splay") == "0":
         log.debug("Trying to obtain the identity immediately, splay is disabled.")
     else:
-        registration_interval = int(cfg.get("rhsmcertd", "auto_registration_interval"))
-        splay_interval: int = random.randint(60, registration_interval * 60)
+        splay_interval: int = 0
+        registration_interval: int = int(cfg.get("rhsmcertd", "auto_registration_interval"))
+        registration_sec_interval: int = int(cfg.get("rhsmcertd", "auto_registration_sec_interval"))
+
+        if registration_sec_interval > 0:
+            # Tha randint() interval begins with 0, because auto-registration v2 has to provide
+            # content as fast as possible, but we still want to use some splay (in seconds).
+            if registration_interval > 0:
+                log.debug(
+                    f"Using auto-registration intervals: {registration_interval} [minutes] "
+                    f"and {registration_sec_interval} [seconds] for random input"
+                )
+                splay_interval: int = random.randint(
+                    0, registration_sec_interval + 60 * registration_interval
+                )
+            else:
+                log.debug(
+                    f"Using auto-registration interval: {registration_sec_interval} [seconds] "
+                    "for random input"
+                )
+                splay_interval: int = random.randint(0, registration_sec_interval)
```

Comment:
Perhaps we could also try `random.seed()` here, to get better RNG? I think @m-horky proposed the cloud instance id from the metadata document

Reply:
Implemented in another PR: https://github.com/candlepin/subscription-manager/pull/3575
in this commit: https://github.com/candlepin/subscription-manager/pull/3575/commits/4c67754d3e18c3f60a78369641c73e6e256d765e